Assets/Scripts/Flock.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flock : MonoBehaviour
+{
+    [SerializeField] private FlockAgent agentPrefab;
+    List<FlockAgent> agents = new List<FlockAgent>();
+    [SerializeField] private FlockBehaviour behaviour;
+
+    [Range(1, 500)]
```

Comment:
what does this range thingy do? :3

Reply:
They add sliders on the editor 
![image](https://user-images.githubusercontent.com/43484130/112241273-505c4b00-8c20-11eb-9883-44266507cbf1.png)